### ISSUE TITLE
Enumerating enums from generated models

### ISSUE THREAD

**Issue by tim-a17** [go-swagger/go-swagger#2292]
## Problem statement

Is it possible to enumerate the generated enums?  Otherwise we end up having to hard code the enums in the code in addition to the Swagger spec.

## Swagger specification

```
swagger: '2.0'
info:
  version: '2.0'
  title: list.yml

paths:
  /:
    get:
      produces:
        - text/plain
      responses:
        200:
          description: returns list
          schema:
            type: array
            items:
              $ref: "#/definitions/list"

definitions:
  list:
    type: string
    enum:
      - LIST_ONE
      - LIST_TWO
      - LIST_THREE

```
## Steps to reproduce

swagger generate server --exclude-main -A list -t gen -f ./list.yml

observe the generated gen/models/list.go file.  The listEnum variable is not public and there are no methods to access it.

## Environment
swagger version: 0.23.0 
go version: 1.14.1  
OS:   Ubuntu 20.04

**Comment by fredbi** [comment]
Yes good point. Feel free to post a PR to expose the generated list.
